src/ui/types/company-details-response.d.ts

```diff
@@ -0,0 +1,8 @@
+interface CompanyDetailsResponse {
+  tradingName: string;
+  tradingAddress: string;
+  companyWebsite: string;
+  phoneNumber: string;
+}
+
+export { CompanyDetailsResponse };
```

Comment:
This doesn't seem to be used anywhere? Seems this should be in the `application` type which will eventually be used when this area is hooked up with the DB/API

Reply:
removed